py2many/astx.py

```diff
@@ -0,0 +1,45 @@
+import ast
+from py2many.clike import LifeTime
```

Comment:
create a new module for this and any other similar enums?

or move Lifetime here?